Frankfurt/FSUor16.xml

```diff
@@ -5,7 +5,7 @@ schematypens="http://relaxng.org/ns/structure/1.0"
     <teiHeader xml:base="https://betamasaheft.eu/">
         <fileDesc>
             <titleStmt>
-                <title>Prayers, Maṣḥafa falasfā ṭabibān, Hāymānotu la-qǝddus ʿābiyy māri Yāʿqob ʾƎlbarādʿi</title>
+                <title>Prayers, <foreign xml:lang="gez">Maṣḥafa falasfā ṭabibān</foreign>, <foreign xml:lang="gez">Hāymānotu la-qǝddus ʿābiyy māri Yāʿqob ʾƎlbarādʿi</foreign></title>
```

Comment:
You preferred fal**ā**sfā (with 'long' **ā** in the middle) in the LIT records in https://github.com/BetaMasaheft/Works/pull/1322/files and in other records in this branch.